components/cycles/CycleOrder.jsx

```diff
@@ -0,0 +1,128 @@
+   paddingLeft: 15px;
+   paddingTop: 5px;
+`;
+
+const SubTitle = styled(CycleTitle)`
+    font-size: 16px;
+    paddingBottom: 5px;
+    font-family: 'Montserrat_600SemiBold';
+`;
+
+const WorkoutViewText = styled.Text`
+    color: #FFFFFF;
+    font-family: 'Montserrat_500Medium';
+    font-size: 12px;
+    paddingTop: 2px;
+    paddingBottom: 4px;
+    width: 33%;
+    textAlign: center;
+`;
+
+const RowHeader = styled.View`
+    flexDirection: row;
+    alignContent: center;
+    font-family: 'Montserrat_500Medium';
+    justifyContent: space-evenly;
+    width:100%;
+`;
+
+const RowContent = styled.View`
+    flexDirection: row;
+    alignContent: center;
+    justifyContent: space-evenly;
+`;
+
+const CycleOrderCard = (props) => {
+  const [showDetailed, setshowDetailed] = React.useState(false);
+  const [icon, setIcon] = React.useState('chevron-down');
+
+  const {
+    name, muscleGroups, color, exercises,
+  } = props;
+  const FullBody = styled.View`
+        width: 90%;
+        margin: auto;
+        background-color: ${color};
+        borderRadius: 10px;
+        box-shadow: 1px 5px 2px gray;
+        marginBottom: 10px;
+        marginTop: 10px;
+        padding: 10px;
+    `;
+
+  const ExercisesView = (items) => {
+    const workoutItems = items.items;
+    return (
+      <View style={{ flexDirection: 'column', justifyContent: 'space-evenly' }}>
+        <RowHeader>
+          <SubTitle>Exercise</SubTitle>
+          <SubTitle>Sets x Reps</SubTitle>
+          <SubTitle>Previous</SubTitle>
+        </RowHeader>
+        <FlatList
+          data={workoutItems}
+          keyExtractor={(item) => item.id}
+          renderItem={({ item }) => (
+            <RowContent>
+              <WorkoutViewText style={{ marginLeft: 10 }}>{item.id}</WorkoutViewText>
+              <WorkoutViewText>{`${item.sets.length}x${item.sets[0].reps}`}</WorkoutViewText>
+              <WorkoutViewText>{`${item.sets[0].weight} lbs.`}</WorkoutViewText>
+            </RowContent>
+          )}
+        />
+      </View>
+    );
+  };
+
+  function showDetail() {
+    if (icon == 'chevron-down') {
+      setIcon('chevron-up');
+    } else {
+      setIcon('chevron-down');
+    }
+    setshowDetailed(!showDetailed);
+  }
+
+  return (
+    <FullBody>
+      <CycleTitle>{name}</CycleTitle>
+      <IconButton
+        icon={icon}
+        color="white"
+        size={25}
+        onPress={showDetail}
+        style={{ position: 'absolute', alignSelf: 'flex-end', marginTop: 0 }}
+      />
+      {!showDetailed && <SubTitle>{muscleGroups}</SubTitle>}
+      {showDetailed && <ExercisesView items={exercises} />}
+    </FullBody>
+  );
+};
+
+CycleOrderCard.propTypes = {
+  name: PropTypes.string,
+  muscleGroups: PropTypes.string,
+  color: PropTypes.string,
+  exercises: PropTypes.array,
+};
+
+CycleOrderCard.defaultProps = {
+  name: 'Cycle Name',
+  muscleGroups: '',
+  color: '#CAB0FF',
+  exercises: [],
+};
+
+export default CycleOrderCard;
```

Comment:
Can we rename this file to something more descriptive, maybe like "ExpandableCycleCard"